bitnami/mongodb/templates/standalone/dep-sts.yaml

```
@@ -278,79 +251,36 @@ spec:
             {{- end }}
           {{- end }}
           ports:
-            - name: mongodb
-              containerPort: 27017
+          {{- range keys .Values.containerPorts }}
+            - name: {{ . | quote }}
+              containerPort: {{ get $.Values.containerPorts . }}
+          {{- end }}
```

```suggestion
          {{- range $name, $port := .Values.containerPorts }}
            - name: {{ $name | quote }}
              containerPort: {{ $port }}
          {{- end }}
```

---

Actually, I don't like managing `containerPorts` this way, since overwritting the object `containerPorts` may delete other values that are not optional.

In this case, I would suggest adding `extraContainerPorts`:
```suggestion
            - name: mongodb
              containerPort: {{ .Values.containerPorts.mongodb }}
            {{- if .Values.extraContainerPorts }}
            {{- include "common.tplvalues.render" (dict "value" .Values.extraContainerPorts "context" $) | nindent 12 }}
            {{- end }}
```

This way, we have `containerPorts.*` in case mandatory ports are added in the future, and `extraContainerPorts` for users custom ports.

---

My bad, that code shouldn't be pushed. Sorry about that